dissect/target/helpers/protobuf.py

```diff
@@ -33,7 +33,7 @@ def decode_varint(stream: BinaryIO) -> int:
     i = 0
     while True:
         byte = stream.read(1)
-        result |= (int.from_bytes(byte) & 0x7F) << (i * 7)
+        result |= (int.from_bytes(byte, byteorder="big") & 0x7F) << (i * 7)
```

Comment:
```suggestion
        result |= (byte[0] & 0x7F) << (i * 7)
```
You're only reading a single byte so no need for an expensive `from_bytes` call.